From 0d86370032070da44df6341181463c74467af37c Mon Sep 17 00:00:00 2001
From: Kedareswara rao Appana <appana.durga.rao@xilinx.com>
Date: Mon, 18 Jul 2016 12:53:35 +0530
Subject: [PATCH 186/188] net: phy: xilinx_phy: Fix compilation warning in the
 driver

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

This commit be01da72b1b8 ("phy: Centralize setting driver module owner")
Centralizes setting driver module owner into the phy core code.

This patch removes the module owner field from the driver as
phy core doing the same.

Signed-off-by: Kedareswara rao Appana <appanad@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
(cherry picked from commit 6d3a88612e9814b6604442ec92cfe0330c0a3f0f)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/phy/xilinx_phy.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/phy/xilinx_phy.c b/drivers/net/phy/xilinx_phy.c
index 1985949..f152f3f 100644
--- a/drivers/net/phy/xilinx_phy.c
+++ b/drivers/net/phy/xilinx_phy.c
@@ -134,7 +134,6 @@ static struct phy_driver xilinx_drivers[] = {
 		.read_status = &xilinxphy_read_status,
 		.resume = &genphy_resume,
 		.suspend = &genphy_suspend,
-		.driver = { .owner = THIS_MODULE },
 	},
 };
 
-- 
2.0.2

